src/miner.cpp

```diff
@@ -19,7 +19,7 @@ using namespace std;
 // BitcoinMiner
 //
 
-unsigned int nMinerSleep;
+volatile unsigned int nMinerSleep;
```

Comment:
This variable is not changed after stake miner thread is started. Without the volatile modifier optimizer prevents me from changing it with debugger at run-time. It is just for debugger.